src/css/styles/_c-btn.scss

```diff
@@ -0,0 +1,38 @@
+.c-btn {
+  display: inline-block;
+  padding: 1.2rem 4rem;
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: $col-bg;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
+  background-color: $col-blue;
+  border: 0;
+  border-radius: 2rem;
+  transition-duration: 0.3s ease-in-out;
+
+  &:hover {
+    text-decoration: none;
+    background-color: $col-dark-blue;
+  }
+
+  &--green {
+    background-color: $col-green;
+
+    &:hover {
+      background-color: $col-dark-green;
+    }
+  }
+
+  &--sm {
+    padding: 0.5rem 1rem !important;
+    font-size: 0.8rem !important;
```

Comment:
We do we need `!important`?

Reply:
Yes, if we hold on to idea of media-queries at the end of the file and BEM guidelines.
As `c-btn--sm` is only a modifier (ergo can't exist alone), the further media-query styling for `c-btn` overwrites its properties.